src/Tools/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -0,0 +1,67 @@
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\AzureBlobEgressProviderOptions.cs" Link="ExternalRefs\AzureBlobEgressProviderOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\AuthenticationOptions.cs" Link="ExternalRefs\AuthenticationOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\GlobalCounterOptions.cs" Link="ExternalRefs\GlobalCounterOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\GlobalCounterOptionsDefaults.cs" Link="ExternalRefs\GlobalCounterOptionsDefaults.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\CorsConfigurationOptions.cs" Link="ExternalRefs\CorsConfigurationOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\DiagnosticPortOptions.cs" Link="ExternalRefs\DiagnosticPortOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\DiagnosticPortOptionsDefaults.cs" Link="ExternalRefs\DiagnosticPortOptionsDefaults.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\DiagnosticPortConnectionMode.cs" Link="ExternalRefs\DiagnosticPortConnectionMode.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\EgressOptions.cs" Link="ExternalRefs\EgressOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\MetricsOptions.cs" Link="ExternalRefs\MetricsOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\MetricsOptionsDefaults.cs" Link="ExternalRefs\MetricsOptionsDefaults.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\MonitorApiKeyOptions.cs" Link="ExternalRefs\MonitorApiKeyOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\FileSystemEgressProviderOptions.cs" Link="ExternalRefs\FileSystemEgressProviderOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\StorageOptions.cs" Link="ExternalRefs\StorageOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\ProcessFilterOptions.cs" Link="ExternalRefs\ProcessFilterOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\IEgressProviderCommonOptions.cs" Link="ExternalRefs\IEgressProviderCommonOptions.cs" />
+    <Compile Include="..\..\Microsoft.Diagnostics.Monitoring.Options\OptionsDisplayStrings.Designer.cs" Link="ExternalRefs\OptionsDisplayStrings.Designer.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="$(AzureIdentityVersion)" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="$(AzureStorageBlobsVersion)" />
+    <PackageReference Include="Azure.Storage.Queues" Version="$(AzureStorageQueuesVersion)" />
+    <PackageReference Include="System.CommandLine" Version="$(SystemCommandLineVersion)" />
```

Comment:
This doesn't seem to be used

Reply:
I meant to add in the commandline parsing from another local copy of the test app. I copied that in and this ref is now used.